unittests/test-contracts/test_api_multi_index/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 if( EOSIO_COMPILE_TEST_CONTRACTS )
-   include_directories( ${Boost_INCLUDE_DIRS} )
-   add_executable( test_api_multi_index test_api_multi_index.cpp )
+   add_contract( test_api_multi_index test_api_multi_index.cpp )
```

Comment:
Missing the contract name, change this to `add_contract( test_api_multi_index test_api_multi_index test_api_multi_index.cpp )`.